lib/rspec/core/memoized_helpers.rb

```diff
@@ -483,9 +483,9 @@ def subject!(name=nil, &block)
       def self.module_for(example_group)
         get_constant_or_yield(example_group, :LetDefinitions) do
           mod = Module.new do
-            include Module.new {
+            include(Module.new {
```

Comment:
IMO, it's a little weird to use parens on `include` calls.  We don't normally do that.  Why here?

Reply:
the rubocop warning was that the block was ambiguous:

```
some_method param { }
```

Does the block belong to the method or the param? That seemed legitimate enough to me to add brackets, but don't feel strongly.